### DISCUSSION TITLE
bigWeb/node/standard-node-cli

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#227] [Announcements]
# bigWeb/node/standard-node-cli

只使用 Node.js 的标准库，不安装任何外部依赖，写一个命令行工具

https://sugarat.top/bigWeb/node/standard-node-cli.html

**Comment by recoluan** [comment]
最近搞了一个脚手架功能集成的小工具，可以叫做开发脚手架的脚手架，看一下有没有兴趣呢 https://github.com/revilijs/revili

**Reply by ATQQ** [reply]
你这个和我做的自己用的cli好像哈哈

**Reply by ATQQ** [reply]
https://github.com/ATQQ/tools/tree/main/packages/cli/dynamic-cli/core

我感觉是想法差不多，指令做成插件可自由的插拔

<img width="382" alt="image" src="https://github.com/ATQQ/sugar-blog/assets/42485491/9662e3e2-57f6-4203-900b-f075df0d0706">

**Reply by recoluan** [reply]
说明这方面应该还是有诉求的哈
